code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -381,7 +381,7 @@ var/global/list/parasites = list() //all currently existing/living guardians
 	var/list/possible_guardians = list("Chaos", "Standard", "Ranged", "Support", "Explosive", "Lightning", "Protector", "Charger", "Assassin")
 	var/random = TRUE
 	var/allowmultiple = 0
-	var/allowling = 1
```

Comment:
If we're going to  fight about TRUE/FALSE vs 1/0 we will be in this codebase for a while.